### PR TITLE
(master) Update docs feature classifications

### PR DESCRIPTION
**Pull request by jlin27**
Update the following feature classifications in docs to align with the changes:
1. [High Level Autograd APIs](https://pytorch.org/docs/stable/autograd.html#functional-higher-level-api): Beta (was experimental) 
2. [Eager Mode Quantization](https://pytorch.org/docs/stable/quantization.html): Beta (was experimental)
3. [Named Tensors](https://pytorch.org/docs/stable/named_tensor.html): Prototype (was experimental)
4. [TorchScript/RPC](https://pytorch.org/docs/stable/rpc.html#rpc): Prototype (was experimental)
5. [Channels Last Memory Layout](https://pytorch.org/docs/stable/tensor_attributes.html#torch-memory-format): Beta (was experimental)
6. [Custom C++ Classes](https://pytorch.org/docs/stable/cpp_index.html): Beta (was experimental)
7. [Torch.Sparse](https://pytorch.org/docs/stable/sparse.html): Beta (was experimental)

### CODE REVIEW

**Review by jspisak — CHANGES_REQUESTED** [review]
The RPC edit isnt the right one. 
it should be the "TorchScript support in RPC is experimental and subject to change."

File: docs/source/rpc.rst [hunk 1]
```diff
@@ -100,7 +100,7 @@ applications can always explicitly move the input tensors to CPU on the caller
 and move it to the desired devices on the callee if necessary.
 
 .. warning::
-  TorchScript support in RPC is experimental and subject to change.
+  TorchScript support in RPC is beta and subject to change.
```
Comment (jlin27) [inline docs/source/rpc.rst:103]: @jspisak - Is this beta or prototype? This doc shows prototype:https://fb.quip.com/7fGEAhAY5Etz

Comment (jspisak) [reply docs/source/rpc.rst:103]: Yes, the TorchScript/RPC is a prototype

Comment (jspisak) [reply docs/source/rpc.rst:103]: Can we also add torch.sparse to this PR as well?  https://pytorch.org/docs/stable/sparse.html?highlight=experimental#

**Review by jspisak — APPROVED** [review]

**Review by jspisak — APPROVED** [review]

**Pull request closed without merging**